webclient/src/components/AppSearchBar.vue

```diff
@@ -7,6 +7,14 @@ import type { SearchResponse } from "@/codegen";
 import { useFetch } from "@/utils/fetch";
 
 export default {
+  mounted() {
+    window.addEventListener("keydown", (e) => {
+      if (e.key === "/") {
+        e.preventDefault();
+        document.getElementById("search")?.focus();
+      }
+    })
```

Comment:
> I think only choosing `s` or `/` are fine.

GitHub, for example, supports both. I don't see a downside to include `(e.key === "s" || e.key === "/")` 

> Also the placeholder text of the input bar should be changed to something like `Press / to Search`.

I don't think, that messing with the placeholder is the right thing to do.

A `placeholder` should (according to https://www.w3schools.com/tags/att_input_placeholder.asp):
> The `placeholder` attribute specifies a short hint that describes the expected value of an input field

We should probably have a question mark button in the search bar (like GitHub, but showing more functionaility) which if hovered over [opens a tooltip](https://picturepan2.github.io/spectre/components/tooltips.html).
The reason I like this solution more than your solution is, that this way we could also explain what query features we support in the future (see #106)

Reply:
I suggest adding the functionality for the focusing itself in this PR and then we could open another for the way we want to inform the user about it. `s` and `/` or _only_ `/` is both totally fine with me, my reasoning behind only giving one functionality one key-bind is that it leaves more room for future expansion and otherwise you also introduce unnecessary redundancy.